fenzo-core/src/main/java/com/netflix/fenzo/AutoScaler.java

```diff
@@ -197,6 +199,10 @@ private boolean shouldScaleDown(long now, ScalingActivity prevScalingActivity, A
         return shouldScaleNow(false, now, prevScalingActivity, rule);
     }
 
+    private boolean shouldScaleDownInactive(long now, ScalingActivity prevScalingActivity, AutoScaleRule rule) {
+        return now > (Math.max(activeVmGroups.getLastSetAt(), prevScalingActivity.inactiveScaleDownAt) + rule.getCoolDownSecs() * 1000);
```

Comment:
Is the idea here that if the vmGroup was just disabled recently, then we want to wait for cooldown period before scaling down?